src/pages/Result.tsx

```diff
@@ -37,12 +37,22 @@ const ButtonContainer = styled.div`
   flex-direction:row;
   gap:1rem;
 `;
+const Winner=styled.p`
+font-size: 1.5rem;
+`;
+const WinnerName=styled.h1`
+font-size: 3rem;
+font-weight: bold;
+`;
+interface IcecreamProps{
+  winner:Icecream;
+}
 //게임 시작 후 결과화면
-function Result({ winner }) {
+function Result( { winner} :IcecreamProps) {
   return (
     <ResultContainer>
-      <p style={{ fontSize: "1.5rem" }}>우승자!</p>
-      <h1 style={{ fontSize: "3rem", fontWeight: "bold" }}>{winner.name}</h1>
+      <Winner>우승자!</Winner>
+      <WinnerName>{winner.name}</WinnerName>
```

Comment:
요사이에
```suggestion
function Result( { winner} :IcecreamProps) {
const { name, img } = winner 
  return (
    <ResultContainer>
      <Winner>우승자!</Winner>
      <WinnerName>{name}</WinnerName>
```

구조분해 할당도 사용가능하다는거!-!

Reply:
> 요사이에
> 
> 구조분해 할당도 사용가능하다는거!-!

훨씬 깔끔하다!! 요렇게 수정 완료~~~